packages/react-native-renderer/src/ReactFabric.js

```diff
@@ -45,7 +45,7 @@ const ReactCurrentOwner = ReactSharedInternals.ReactCurrentOwner;
 
 function findHostInstance_DEPRECATED(
   componentOrHandle: any,
-): ?HostComponent<mixed> {
+): ?React$ElementRef<HostComponent<mixed>> {
```

Comment:
I used dollar signs here because these ones are inside the renderer and it doesn't seem to make sense to `import 'react'` at the top of this file. This seems to be consistent with the rest of the files in the React repo.